prow/cmd/hook/main.go

```diff
@@ -110,34 +108,30 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
+	var tokens []string
+	var err error
```

Comment:
I *think* this is unused? In the following:

```go
if err := somefunc(); err != nil {
```

The `err` does not escape the `if` block scope